content/docs/user-guide/components/reference/terrain/layer_spawner.md

```diff
@@ -29,10 +39,9 @@ The [Axis-Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-al
 | **Sub Priority** | Sets the priority of this spawner within the layer. Higher numbers will override lower. | 0 - 10000 | 0 |
 | **Use Ground Plane** | Enable this setting to provide a default ground plane where no terrain is defined. | Boolean | `True` |
 
-
 ## TerrainSpawnerRequestBus
 
-Use the following request functions with the `TerrainSpawnerRequestBus` EBus interface to communicate with Terrain Layer Spawner components of your game.
+The `TerrainSpawnerRequestBus` is an internal EBus used by the terrain system to query the Terrain Layer Spawner settings. It shouldn't generally be needed or used by other systems, since nothing outside the terrain system should need any information about individual Terrain Layer Spawners. However, if a use case should arise, the following request functions on the `TerrainSpawnerRequestBus` EBus interface can be used to query the individual Terrain Layer Spawner components.
```

Comment:
```suggestion
The `TerrainSpawnerRequestBus` is an internal EBus used by the terrain system to query Terrain Layer Spawner settings. It is not generally used by other systems since nothing outside the terrain system should need any information about individual Terrain Layer Spawners. However, if a use case arises, the following request functions on the `TerrainSpawnerRequestBus` EBus interface can be used to query the individual Terrain Layer Spawner components.
```

Reply:
Hm, how about: "Other systems generally do not need to use this EBus since nothing outside the terrain system should need any information about individual Terrain Layer Spawners."

This moves it to active voice and brings back the "hypothetical" bent of the sentence, since the point is more intended to be that new systems won't need it, not that existing systems don't use it.

Reply:
> How about: "Other systems generally do not need to use this EBus since nothing outside the terrain system should need any information about individual Terrain Layer Spawners."

I like it.